src/dvclive/live.py

```diff
@@ -191,6 +198,78 @@ def log_plot(self, name, labels, predictions, **kwargs):
         data.dump(val, self._step, **kwargs)
         logger.debug(f"Logged {name}")
 
+    def log_params(
+        self, params: Dict[str, ParamsLike], name: str = PARAMS_DEFAULT_NAME
+    ):
+        """Saves the given set of parameters (dict) to json"""
+        if self._resume:
+            logger.debug("resume=True, not removing existing parameters.")
+            return
+
+        output_dir = Path(self.dir) / self.PARAMS_DIR
+        output_dir.mkdir(exist_ok=True)
+        params_path = output_dir / (
+            f"{name}.json" if not name.endswith(".json") else name
+        )
+
+        if params_path.is_file():
+            with open(params_path, "r", encoding="utf-8") as fobj:
+                loaded_params = json.load(fobj)
+
+            for param_name, param_value in params.items():
+                if param_name in loaded_params:
+                    raise ParameterAlreadyLoggedError(
+                        param_name, param_value, loaded_params[param_name]
+                    )
+            params = {**params, **loaded_params}
+
+        self._params.setdefault(name, {}).update(**params)
+
+        with open(params_path, "w", encoding="utf-8") as f:
+            json.dump(params, f, indent=4, cls=NumpyEncoder)
+
+        logger.debug(
+            f"Logged {len(params.keys())} parameters to {params_path}"
+        )
+
+    def log_param(
+        self,
+        name: str,
+        val: Union[int, float, str, bool],
+        params_set_name: str = PARAMS_DEFAULT_NAME,
```

Comment:
(Product) I am not sure about providing this option. 

I believe we could start by "forcing" a fixed output file of our choice so that the Onboarding workflow gets easier to implement.

We can wait for users to ask for this unless we see a good reason to include the logic as part of the current onboarding proposal (right now feels a little like a deviation from what we have been discussing)

Reply:
That's why this is optional ("forcing" a fixed output). The idea is to let users log parameters to different files to keep a flatter structure in each file. Could get rid of this though.

Reply:
Seems like it deviates from the rest of dvclive where the output structure is fixed relative to the dvclive path, and it might be confusing to document, so my preference is to leave this out until we have a need.

Reply:
Got rid of it